cinema_service/settings.py

```diff
@@ -20,9 +21,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
```

Comment:
Security issue: It's not safe to store the SECRET_KEY in your settings file, especially if it's version controlled. This key should be kept secret. Consider using environment variables to store your secret key.